tests/common/plugins/sanity_check/__init__.py

```diff
@@ -14,10 +14,13 @@
 from tests.common.plugins.sanity_check.recover import neighbor_vm_restore
 from tests.common.plugins.sanity_check.constants import STAGE_PRE_TEST, STAGE_POST_TEST
 from tests.common.helpers.assertions import pytest_assert as pt_assert
+from tests.common.config_reload import config_reload
 
 logger = logging.getLogger(__name__)
 
 SUPPORTED_CHECKS = checks.CHECK_ITEMS
+CONFIG_DB = "/etc/sonic/config_db.json"
+GOLDEN_CONFIG = "/etc/sonic/golden_config_db.json"
```

Comment:
When do we create this golden config?

Reply:
One method to backup a golden config for later reference is during "testbed-cli.sh deploy-mg".

Reply:
> When do we create this golden config?

Golden config is generated by NDM team, and they will put the golden_config_db.json under the folder `/etc/sonic` on dut. 

refer to:
https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/214923/SONiC-Config-Flows?anchor=3.3.-ndm-service-generate-golden-config